File: src/main/java/com/bajdcc/LALR1/interpret/os/user/routine/URPC.java
package com.bajdcc.LALR1.interpret.os.user.routine;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【用户态】行为树示例：计算机网络模拟
 *
 * @author bajdcc
 */
public class URPC implements IOSCodePage {
	@Override
	public String getName() {
		return "/usr/p/pc";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/user/routine/URPC.kt
package com.bajdcc.LALR1.interpret.os.user.routine

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【用户态】行为树示例：计算机网络模拟
 *
 * @author bajdcc
 */
class URPC : IOSCodePage {
    override val name: String
        get() = "/usr/p/pc"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
